### ISSUE TITLE
TypeError: _store.summaryCharts[level] is undefined

### ISSUE THREAD

**Issue by davidlenz** [jina-ai/dashboard#27]
Hi, thanks for the fabulous Jina!

**TL;DR:**
When i try to access them logs on the dashboard i get the following error:

![grafik](https://user-images.githubusercontent.com/25869733/88395966-2406db00-cdc2-11ea-8abb-48f13c475203.png)

**More Details:**

Self-hosted a dashboard and it worked out fine (the `pod_api` line for `mylogserver.yml` is not yet in the docs). The dashboard is hosted on a remote machine and i can access from my local machine. So far so good.

![grafik](https://user-images.githubusercontent.com/25869733/88395068-c3c36980-cdc0-11ea-8143-43ca55f64704.png)

On the same remote machine i am indexing the southpark example. I edited `flow-index.yml` to use logserver.
`root@tmk-0-fra1:~/ex2/examples/southpark-search# cat flow-index.yml`:
```yaml
!Flow
with:
  read_only: true
  port_expose: $JINA_PORT
  logserver: true
  logserver_config: mylogserver.yml
pods:
  reader:
    yaml_path: pods/extract.yml
  splittor:
    yaml_path: pods/craft-split.yml
    replicas: $REPLICAS
    read_only: true
  encoder:
    yaml_path: pods/encode.yml
    replicas: $REPLICAS
    timeout_ready: 1200000
    read_only: true
  chunk_indexer:
    yaml_path: pods/index-chunk.yml
    replicas: $SHARDS
    separated_workspace: true
  doc_indexer:
    yaml_path: pods/index-doc.yml
    needs: gateway
  join_all:
    yaml_path: _merge
    needs: [doc_indexer, chunk_indexer]
    read_only: true
```

`root@tmk-0-fra1:~/ex2/examples/southpark-search# cat mylogserver.yml`:
```yaml
host: 0.0.0.0
port: 5000
endpoints:
  log: /stream/log  # fetching log in SSE stream
  profile: /stream/profile  # fetching profiling log in SSE stream
  yaml: /data/yaml  # get the YAML spec of a flow
  shutdown: /action/shutdown  # shutdown the log server
  ready: /status/ready  # tell if the log server is ready, return 200 if yes
  podapi: /data/api/pod  # get the api spec of the pod
```

run `python3 app.py -t index -n 250` 
![grafik](https://user-images.githubusercontent.com/25869733/88395633-a9d65680-cdc1-11ea-97a5-1e9fd64f4b38.png)

But when i try to access them logs it gives a bunch of erros (they're all the same):
![grafik](https://user-images.githubusercontent.com/25869733/88396263-92e43400-cdc2-11ea-9de4-5127aff1716b.png)

**Comment by imsergiy** [comment]
We were not able to reproduce the issue on our end but we will keep an eye out for this
